src/3D/setup3d.F90

```diff
@@ -799,6 +807,11 @@ subroutine setup(fileName, fileType)
             end do
         end do patchLoop
 
+        ! if there was an error in the BC setup stop
+        if (bcError) then
+            stop
```

Comment:
There is no message printed here (like done with the other later). We should probably add something to alert the user.